action/setup-pacman.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
```

Comment:
SPDX line?

Reply:
Let's fix in a separate PR no? Half the files in mkosi are missing SPDX lines.

Reply:
Fair enough, didn't notice that - just saw this new file without it